feat: Add compliance dashboard

yevgenypats opened the pull request:
#### Summary

This also includes the following fixes:
* check field is a string
* execution_time is set only once and not in every subpolicy
* execution_time uses native PostgreSQL now() instead of data
which makes it work on windows
* make policies use relational paths and now you can execute it from any directory.


NOTE: There is still work to be done in the dashboard and potentially some rough edges but I think this is a good start.

Other ideas: add severity to checks which can make some of the visualizations more interesting.

Use the following steps to ensure your PR is ready to be reviewed

- [ ] Read the [contribution guidelines](../blob/main/CONTRIBUTING.md) 🧑‍🎓
- [ ] Run `go fmt` to format your code 🖊
- [ ] Lint your changes via `golangci-lint run` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation))
- [ ] Update or add tests. Learn more about testing [here](https://docs.cloudquery.io/docs/developers/sdk/testing) 🧪
- [ ] Update the docs by running `go run ./docs/docs.go` and committing the changes 📃
- [ ] If adding a new resource, add [relevant Terraform files](../terraform) in a separate PR 📂
- [ ] Ensure the status checks below are successful ✅

yevgenypats committed:
feat: Add compliance dashboard

This also includes the following fixes:
* check field is a string
* execution_time is set only once and not in every subpolicy
* execution_time uses native PostgreSQL now() instead of data
which makes it work on windows

yevgenypats committed:
Update README to include reference to compliance dashboard

shimonp21 reviewed (changes requested):
Why some places are 'execution_time' and some are 'execution_time'::timestamp?

hermanschaaf committed:
Some fixes to the dashboard

- Add data source row
- Fix colors in Pass/Fail Distribution
- Regenerate screenshot

hermanschaaf commented on policies/README.md:
```diff
@@ -30,3 +30,15 @@ psql -U postgres -c "select * from aws_policy_results" --csv
+
+1. Execute one or more of the above policies.
+2. Add the CloudQuery postgres database as a data source to Grafana (`Configuration -> Data Sources -> Add Data Source`)
+3. Import [dashboards/azure/azure_asset_inventory.json](../dashboards/grafana/compliance.json) into Grafana (`Import -> Upload JSON File`).
```
@yevgenypats I think we should move most of this `Visualization` section to a separate readme under the `dashboards` directory, and link to that section from here (maybe while keeping the screenshot). That way, you can look at the `dashboards` directory on its own and still know how to use it. WDYT?

yevgenypats replied:
Yeah possible. I would still link to Visualization then from this README and also backlink from visualizations to policies.

hermanschaaf replied:
Yup, that's what I had in mind with my comment too 👍

hermanschaaf reviewed (approved):
Looks great :+1: I pushed some updates to the JSON to add a Data Source row and fix the colors of the distributions, which somehow always showed green in my case. Left a few questions, but nothing blocking

yevgenypats commented:
> Why some places are 'execution_time' and some are 'execution_time'::timestamp?

some Postgres edge case in some queries where it couldn't do a cast implicitly 🤷 so it failed and I had to add that.

yevgenypats committed:
Update policies/README.md

Co-authored-by: Herman Schaaf <hermanschaaf@gmail.com>

yevgenypats committed:
Update policies/README.md

Co-authored-by: Herman Schaaf <hermanschaaf@gmail.com>
